examples/time_series/noaa_json.py

```diff
@@ -0,0 +1,74 @@
+"""
+===================================================================
+Plotting of near real time (NRT) data by fetching it from json file
+====================================================================
+
+How to query and visualize nrt data(json format) using pandas and plot
+it using TimeSeries Library
+"""
+
+
+from collections import OrderedDict
+
+import pandas as pd
+
+from astropy import units as u
+
+from sunpy import timeseries as ts
+from sunpy.time import parse_time
+
+data = pd.read_json("https://services.swpc.noaa.gov/json/goes/primary/xrays-7-day.json")
+
+############################################
+# over here the json is transformed and stored as a pandas dataframe
+
+data_short = data[data["energy"] == "0.05-0.4nm"]
+
+##################################################################
+# Filtering the data: A new dataframes are created by filtering the original data.
+# The data_short dataframe contains only the rows where the value of the "energy" column is "0.05-0.4nm"
+
+data_long = data[data["energy"] == "0.1-0.8nm"]
```

Comment:
```suggestion
# The aim here is to display the two seperate energy bands that GOES can measure.
# These being: "0.05-0.4nm" and "0.1-0.8nm". Therefore we will need to filter the data.

# This will get us the short wavelength data.
goes_short = data[data["energy"] == "0.05-0.4nm"]
# This will get us the long wavelength data.
goes_long = data[data["energy"] == "0.1-0.8nm"]
```